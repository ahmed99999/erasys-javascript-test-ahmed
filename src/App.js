import React from 'react';
import './App.css';
import { Route, Redirect, Switch } from 'react-router-dom';
import Users from './components/Users';


function App() {
  return (
    <Switch>
      <Route path="/users" component={Users} />
    </Switch>
  );
}

export default App;
