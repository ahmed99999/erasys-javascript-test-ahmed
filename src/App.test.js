import React from 'react';
import { BrowserRouter, Switch, Route } from 'react-router-dom';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import App from './App';

test('renders learn react link', () => {
  const component = renderer.create(
    <BrowserRouter>
      <App />
    </BrowserRouter>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
  // const linkElement = getByText();
  // console.log(linkElement);
  // expect(linkElement).toBeInTheDocument();
});
