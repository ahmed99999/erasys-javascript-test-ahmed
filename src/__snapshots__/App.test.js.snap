// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
Array [
  <div
    className="Toastify"
  />,
  <div>
    <div
      className="center"
    >
      <div
        className="sc-AxjAm dFebJs"
      >
        <div
          className="sc-AxirZ bSKiQU"
          color="#00bfff"
          size={50}
        />
      </div>
    </div>
  </div>,
]
`;
